### PR TITLE
added teardown for esec to acceptance tests

### PR DESCRIPTION
**Pull request by Phemus**
Adds a python module to remove all users created in security as part of the acceptance test run. runs once after all tests if the environment is integration

### CODE REVIEW

File: esec_teardown/server.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+LOGGER = logging.getLogger(__name__)
+
+# This tells the SSL runtime to trust the provided certificate without credentials
+_create_unverified_https_context = ssl._create_unverified_context
+
+logging.basicConfig(filename="./esec-delete.log", level=logging.DEBUG)
+
+# TODO: Ensure the domain name is in the /windows/system32/drivers/etc/hosts file
+#url = 'https://TST-ESEC-H204.tst.esec.net:8443/routerProject/services/DocumentServices?wsdl'
```
Comment (Kirosoft) [inline esec_teardown/server.py]: remove these comments pls

File: esec_teardown/server.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    return settings
+
+
+def run_delete():
+    LOGGER.info("ESEC delete init")
+    settings = process_input_options()
+    LOGGER.info("Settings from command line: %s" % str(settings))
+    ORGANISATION_ID = settings.organisation_name
+
+    #old account = LR Administrators
```
Comment (Kirosoft) [inline esec_teardown/server.py]: remove this comment

File: esec_teardown/server.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+url = os.getenv('ESEC_URL')
+adminUrl = os.getenv('ESEC_ADMIN_URL')
+userServicesUrl = os.getenv('USER_SERVICE_URL')
+
+client = Client(url)
+adminClient = Client(adminUrl)
+userClient = Client(userServicesUrl)
+
+doTheDelete = False
```
Comment (Kirosoft) [inline esec_teardown/server.py:19]: remove as unused

File: esec_teardown/server.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                try:
+                    if settings.confirm_delete == "True":
+                        res = adminClient.service.removeUser(requestId, user["userId"])
+                        LOGGER.info("Remove user: %s - %s " % (user["userId"],str(res)))
+                    else:
+                        LOGGER.error("Remove user(TEST- NOT DELETED): %s - %s " % (user["userId"],0))
+                except:
+                    msg = str(sys.exc_info())
+                    print(msg)
+                    LOGGER.error(("Error: Removing user: %s" % (user["userId"])))
```
Comment (Kirosoft) [inline esec_teardown/server.py:62]: remove of user should be written to the logs

File: esec_teardown/server.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+import os
+
+
+LOGGER = logging.getLogger(__name__)
+
+_create_unverified_https_context = ssl._create_unverified_context
+
+logging.basicConfig(filename="./esec-delete.log", level=logging.DEBUG)
+
+url = os.getenv('ESEC_URL')
```
Comment (tsangsimon) [inline esec_teardown/server.py]: 3 wasted lines here as it seems your assigning 3 variables that get used only once. might as well initialise the Client with the os.getenv directly.

**Comment by tsangsimon** [comment]
:+1:

**Comment by Kirosoft** [comment]
:+1:

**Pull request merged**